src/components/LeftSidebar/SearchConversationsResults/SearchConversationsResults.vue

```diff
@@ -0,0 +1,314 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import { RecycleScroller } from 'vue-virtual-scroller'
+
+import IconChatPlus from 'vue-material-design-icons/ChatPlus.vue'
+
+import { loadState } from '@nextcloud/initial-state'
+import { t } from '@nextcloud/l10n'
+
+import NcAppNavigationCaption from '@nextcloud/vue/components/NcAppNavigationCaption'
+import NcListItem from '@nextcloud/vue/components/NcListItem'
+
+import AvatarWrapper from '../../AvatarWrapper/AvatarWrapper.vue'
+import ConversationIcon from '../../ConversationIcon.vue'
+import Hint from '../../UIShared/Hint.vue'
+import Conversation from '../ConversationsList/Conversation.vue'
+
+import { ATTENDEE, CONVERSATION, AVATAR } from '../../../constants.ts'
+import { useSettingsStore } from '../../../stores/settings.js'
+import type { Conversation as TypeConversation, ParticipantSearchResult } from '../../../types/index.ts'
+
+const props = defineProps<{
+	searchText: string,
+	conversationsList: Array<TypeConversation>,
+	contactsLoading: boolean,
+	searchResultsListedConversations: Array<TypeConversation>,
+	searchResults: Array<ParticipantSearchResult>,
+}>()
+
+const emit = defineEmits<{
+	(event: 'abort-search'): void,
+	(event: 'create-new-conversation', searchText: string): void,
+	(event: 'create-and-join-conversation', item: TypeConversation): void,
+}>()
+
+const isCirclesEnabled = loadState('spreed', 'circles_enabled')
+const canStartConversations = loadState('spreed', 'start_conversations')
+const settingsStore = useSettingsStore()
+const isCompact = computed(() => settingsStore.conversationsListStyle === CONVERSATION.LIST_STYLE.COMPACT)
+
+// Item's content (avatar) + internal_padding * 2 + external_padding * 2
+const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
+
+// Other sections than joined conversations in the output order
+const sections = [
+	{
+		type: 'user',
+		caption: { type: 'caption', id: 'users_caption', name: t('spreed', 'Users') },
+	},
+	{
+		type: 'group',
+		caption: { type: 'caption', id: 'groups_caption', name: t('spreed', 'Groups') },
+	},
+	{
+		type: 'circle',
+		caption: { type: 'caption', id: 'circles_caption', name: t('spreed', 'Teams') },
+	},
+	{
+		type: 'federated',
+		caption: { type: 'caption', id: 'federated_users_caption', name: t('spreed', 'Federated users') },
+	},
+] as const
+
+type subListType = {
+	user: ParticipantSearchResult[],
+	group: ParticipantSearchResult[],
+	circle: ParticipantSearchResult[],
+	federated: ParticipantSearchResult[],
+}
```

Comment:
Type name should be UpperCase for the first letter and without `Type` (type is a type anyway)